sass/components/_slider-container.scss

```diff
@@ -0,0 +1,102 @@
+.slider-container {
+    background-color: #eeeff0;
+    display: flex;
+    flex-direction: column;
+    padding: 10px 50px;
+    justify-content: center;
+
+    .slider-title {
+        font-size: 1.5em;
+        text-align: center;
+        padding: 50px 50px 20px;
+    }
+
+    .slider {
+        //align-items: center;
+        text-align: center;
+        //justify-content: space-around;
+        width: 60%;
+        margin: auto auto 50px;
+
+        .element-photo {
+            img {
+                display: inline-block;
+                margin-top: -60px;
+                transform: scale(0.7);
+                transition: transform 0.3s ease, margin-left 1s ease;
+                // &:hover {
+                //     transform: scale(1);
+                //     margin-left: 50px;
+                // }
```

Comment:
W css raczej się nie trzyma komentarzy, więc zazwyczaj je możesz usunąć. Ale jeśli celowo chcesz któryś zatrzymać to nie problem